lib/utils/platform.js

```diff
@@ -7,7 +7,7 @@ var STAGE = 'staging';
 var PROD = 'production';
 
 var api = function api(platform) {
-  if (platform === DEV) return 'http://localhost:9100';
+  if (platform === DEV) return process.env.REACT_APP_API ? process.env.REACT_APP_API : 'http://localhost:9100';
```

Comment:
I can't comment on the particular lines below (lines 19 and 28), but I think we need to have similar env vars for both the `applyUI` and `marketingUI`s.